src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -0,0 +1,195 @@
+import csv
+import getopt
+import json
+import os
+import os.path
+import serialobj
+import sys
+sys.path.append(os.getenv('HELIX_SCRIPT_ROOT'))
+import helix.logs
+
+log = helix.logs.get_logger()
+
+def readCSV(csvFile):
+    log.info('Reading '+csvFile)
+    if not os.path.exists(csvFile):
+        raise Exception(csvFile+' could not be found for conversion')
+
+    csvdict = dict()
+    with open(csvFile, 'rb') as csvfile:
+        reader = csv.reader(csvfile)
```

Comment:
I have tried adding some commenting but the structure is, Each run has suites, each suite has tests and each test has a number of results, (in other words dict->dict->dict->list)